pkg/controller/instanceset/instance_util.go

```diff
@@ -91,13 +88,15 @@ type instance struct {
 func ParseParentNameAndOrdinal(s string) (string, int) {
```

Comment:
How about adding a benchmark test to this method?

Reply:
fixed by [9e4d782](https://github.com/apecloud/kubeblocks/pull/8045/commits/9e4d782a816bdb33aa94dcd3f982b1c3c3282dec)